src/test/java/com/wikia/webdriver/common/contentpatterns/WikiTextContent.java

```diff
@@ -55,13 +58,13 @@
   public static final String BLUELINK_TEXT = LINK_PREFIX + PageContent.INTERNAL_LINK + LINK_POSTFIX;
   public static final String
       EXTERNAL_LINK_TEXT =
-      LINK_PREFIX + PageContent.EXTERNAL_LINK + LINK_POSTFIX;
+      EXT_LINK_PREFIX + PageContent.EXTERNAL_LINK + EXT_LINK_POSTFIX;
   public static final String REDLINK_TEXT = LINK_PREFIX + PageContent.REDLINK + LINK_POSTFIX;
   public static final String
       REDIRECT_LINK_TEXT =
       LINK_PREFIX + PageContent.REDIRECT_LINK + LINK_POSTFIX;
 
-	private WikiTextContent() {
+  private WikiTextContent() {
```

Comment:
It's a private Constructor. All of the worker classes have them now. The reason is that if there's no Constructor method defined, there will be a default public Constructor available. This would prevent ppl from accidentally creating an object.